api/src/main/resources/_core/database/mysql/dimensions/dim_obs_group/sp_mamba_dim_obs_group_create.sql

```diff
@@ -0,0 +1,21 @@
+-- $BEGIN
+
+CREATE TABLE mamba_dim_obs_group
+(
+    id                            INT          NOT NULL AUTO_INCREMENT,
+    obs_group_concept_id          INT          NOT NULL,
+    obs_group_name                VARCHAR(255) NOT NULL, -- should be the concept name of the obs
```

Comment:
@GOmare this comment you left here is because you are still working on the name as you indicated during the call we had this morning?